examples/render/contour.yaml

```diff
@@ -276,6 +1115,7 @@ spec:
   completions: 1
   backoffLimit: 1
 ---
+---
```

Comment:
I'm worried about all of these double ---'s

Reply:
I did do a check and they don't cause any errors with the kubectl yaml parsing. I figured it was better to have double `---` instead of not enough.

Reply:
I can change it if you like though.

Reply:
I've changed everything to have `---` only at the start of the file, that should remove the double ups.